ls1intum/Artemis
# [Bugfix] Gitlab Jenkins Api Tests for Programming Exercises

### Checklist

### Motivation and Context


The Gitlab/Jenkins Api tests were for programming exercises when a member was added into a repository.

### Description

Rewrote the function so that the test doesn't fail.

### Steps for Testing

The functionality hasn't been changed. But if testing should be made, the same steps apply as in this [closed pr](https://github.com/ls1intum/Artemis/pull/2929).

### Test Coverage



ZipFileService.java 93%
GitlabService.java 43%
